x/staking/keeper/slash_test.go

```diff
@@ -669,7 +669,7 @@ func TestSlashValidatorAtCurrentHeightWithSlashingProtection(t *testing.T) {
 	// increase the block number to be able to get the reward
 	ctx = app.BaseApp.NewContext(false, tmproto.Header{Height: app.LastBlockHeight() + 1})
 	// now slash based on the current power
-	app.StakingKeeper.Slash(ctx, vaConsAddr, ctx.BlockHeight(), power, fraction)
+	app.StakingKeeper.Slash(ctx, vaConsAddr, ctx.BlockHeight(), power, fraction, types.InfractionEmpty)
```

Comment:
I don't know why they added this argument in and later removed it, from the implementation it looks like it goes unused so I can simply use the default InfractionEmpty value. https://github.com/cosmos/cosmos-sdk/pull/14363